src/bitops.c

```diff
@@ -784,14 +891,76 @@ void bitopCommand(client *c) {
                     output |= byte;
                     skip = (output == 0xff);
                     break;
-                case BITOP_XOR: output ^= byte; break;
+                case BITOP_XOR:
+                    output ^= byte;
+                    break;
+
+                /* For DIFF, DIFF1 and ANDOR we compute the disjunction of all key arguments except
+                 * the first one. After that we do their respective bit op on said first arg and that
+                 * disjunction. */
+                case BITOP_DIFF:
+                case BITOP_DIFF1:
+                case BITOP_ANDOR:
+                    disjunction |= byte;
+                    skip = (disjunction == 0xff);
+                    break;
+
+                /* BITOP ONE dest key_1 [key_2...]
+                 * If dest[i] is the i-th bit of dest then:
+                 * dest[i] == 1 if and only if there is j such that key_j[i] == 1 and key_n[i] == 0
+                 * for all n != j.
+                 * In order to compute that on each step we track which bits were seen in more than
+                 * one key and store that in a helper variable. Then the operation is just XOR but on
+                 * each step we nullify the bits that are set in the helper.
+                 * Logically, this operation is the same as nullifying the helper bits only once at
+                 * the end, but performance-wise it had no significant benefit and makes the code
+                 * only more unclear.
```

Comment:
I see that in this file, most lines try to confirm 80 characters line limit. Let's try to follow it. Could you wrap comments please?